index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <!--<p>Hello world, we are software developers</p>-->
```

Comment:
Apprieciate this work , please can you remove the unnesseacery comments in your code base

Reply:
Nice job making use of the github inline comment feature👍. To make our comments better, let's try to give reasons why we are requesting a change😉. don't forget, it might be clear to you but not to others😉. to make your comments more descriptive and easy to understand, I suggest you state why you are requesting a change👍.

you can do well to edit this comment to implement the change😉.